### ISSUE TITLE
[feature request] google frequency list as an output option

### ISSUE THREAD

**Issue by omega3** [soimort/translate-shell#368]
When I translate 
https://translate.google.pl/?hl=pl&tab=wT#view=home&op=translate&sl=auto&tl=pl&text=conclude
I get this output:

```
trans conclude
conclude
/kənˈklo͞od/

wyciągnąć wniosek

conclude – definicje
[ English -> Polski ]

czasownik
    zakończyć
        end, complete, finish, terminate, conclude, wind up
    wnioskować
        conclude, reason, gather, argue, collect
    wywnioskować
        deduce, conclude, infer, understand, gather, induce
    sfinalizować
        finalize, conclude, sign, settle, button up
    konkludować
        conclude, infer
    dokończyć
        complete, finish, end, finish up, conclude, do away
    finalizować
        conclude, settle

conclude
    wyciągnąć wniosek, wnioskować, zakończyć, wywnioskować, sfinalizować


```
This:  

`wyciągnąć wniosek, wnioskować, zakończyć, wywnioskować, sfinalizować`

differs from this (frequency list):

```
     zakończyć
        end, complete, finish, terminate, conclude, wind up
    wnioskować
        conclude, reason, gather, argue, collect
    wywnioskować
        deduce, conclude, infer, understand, gather, induce
    sfinalizować
        finalize, conclude, sign, settle, button up
    konkludować
        conclude, infer
    dokończyć
        complete, finish, end, finish up, conclude, do away
    finalizować
        conclude, settle

```

The question is why?

Is there any way (a small change in coding) that can create something like this:

zakończyć / wnioskować / wywnioskować / sfinalizować / konkludować / dokończyć / finalizować

from the above (frequency) list:
https://i.imgur.com/idMbMKa.png

Actually, frequency list has more meanings. **Could you add it as an option?**
Basically I would like to have all meanings in one line separated by /.

I really badly need it for my Anki flashcards:
https://i.imgur.com/hsZvTOX.png

Or maybe someone can tell me how to do it as a script.

**Comment by soimort** [comment]
The last line of the output
```
wyciągnąć wniosek, wnioskować, zakończyć, wywnioskować, sfinalizować
```
is *not* related to frequency at all. It's the "suggestions for alternative translation" you will get when you ***click*** on the translation on Google's web interface, in the original order given by Google Translate:

![Screenshot from 2020-09-04 15-02-02](https://user-images.githubusercontent.com/342945/92242658-6b53b180-eec0-11ea-8418-6eec0859fd2c.png)

There is currently no plan to add an extra option for printing the `/`-separated frequency list, because the dictionary output already has everything in order and can be filtered by an external script (written in shell, sed, awk, etc.) easily. You may also use the `-dump` option to get the raw JSON output (which can be processed in pretty much any way as you like).
